src/js/pensions/containers/ConfirmationPage.jsx

```diff
@@ -54,8 +55,18 @@ class ConfirmationPage extends React.Component {
               <strong>Date received</strong><br/>
               <span>{moment(form.submission.submittedAt).format('MMM D, YYYY')}</span>
             </li>
+            <li>
+              <strong>Confirmation number</strong><br/>
+              <span>{response.confirmationNumber}</span>
+            </li>
+            <li>
+              <strong>Your claim was sent to</strong><br/>
+              <address className="schemaform-address-view">{regionalOffice.map((line, index) =>
+                <p key={index}>{line}</p>)}</address>
```

Comment:
I learned the other day that eslint doesn't like us using `index` as `key`. Says it [isn't unique enough](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-array-index-key.md) (basically). I don't mind it--especially not here--but it's a consideration. :man_shrugging: